app/translations/dutch.js

```diff
@@ -0,0 +1,396 @@
+  receivePageLabel: 'Ontvang assets',
+  receiveYourAddressTabLabel: 'JOUW ADRES',
+  receiveCopyCodeButton: 'Copy Code Afbeelding',
+  receiveDisclaimer:
+    'Stuur alleen assets die <b>compatibel zijn met de NEO-blockchain (NEO, GAS, etc.)</b>. Het verzenden van andere assets zal resulteren in permanent verlies.',
+  receiveRequestTabAssets: 'VERZOEK ASSETS',
+  recieveWhyUseQRLabel: 'Waarom een QR-Code gebruiken??',
+  receiveQRExplanation:
+    '<p>Ooit assets naar het verkeerde adres gestuurd vanwege een foutief karakter in het wallet-adres?</p><p>Zo niet, dan heb je mazzel - maar het gebeurt veel te vaak.</p><p>Hier bij CoZ willen we ervoor zorgen dat mensen die jou betalen, het juiste adres krijgen. J kunt een QR-code genereren om assets aan te vragen, zodat ze jou kunnen helpen. </p> <p> Elke code die je genereert, bevat uw openbare wallet-adres, een aantal en een referentie - allemaal door jou ingesteld. </p>',
+}
+
+const REQUEST_ASSETS = {
+  requestAssetLabel: 'ASSET',
+  requestAssetAmountLabel: 'Aantal',
+  requestAssetAmount: 'AANTAL',
+  requestAssetDepositLabel: 'SORT IN DEZE WALLET',
+  requestAssetRefLabel: 'REFERENTIE',
+  requestAssetRefPlaceholder: 'Voeg een notitie toe...',
+  requestAssetQRButton: 'Genereer QR-Code',
+  requestAssetYourQRHeader: 'Jou QR Code',
+  requestAssetsPaymentDetails: 'BETAALVERZOEK DETAILS',
+  requestAssetsYourQRLabel: 'Jouw QR-Code',
+  requestAssetsRefLabel: 'REFERENTIE',
+  requestAssetsAddressLabel: 'ADRES',
+  requestAssetsAmountLabel: 'AANTAL',
+  requestAssetsAssetLabel: 'ASSET',
+}
+
+const TRANSACTION_FEE = {
+  transactionFeeQuestion: 'PRIORITEER JOUW TRANSACTIE MET EEN FEE?',
+  fast: 'Snel',
+  faster: 'Sneller',
+  fastest: 'Snelst',
+  sendWithFee: 'Verstuur {itemCount, plural, one {Asset} other {Assets}} Met fee',
+  sendWithoutFee:
+    'Verstuur {itemCount, plural, one {Asset} other {Assets}} Zonder fee',
+  Asset: 'Asset',
+  assets: 'Assets',
+}
+
+const SEND = {
+  sendPageLabel: 'Verstuur Assets',
+  sendImport: 'Importeren',
+  sendEnterQRCode: 'Voer QR-Code in',
+  sendAdd: 'Voeg ontvanger toe',
+  sendAssetLabel: 'ASSET',
+  sendAmountLabel: 'AANTAL',
+  sendAddressLabel: 'ONTVANGER ADRES',
+  sendAddressPlaceholder: 'Voeg wallet toe of selecteer contact',
+  sendTranfer: 'TRANSACTIE',
+  sendMaxAmount: 'MAX',
+  sendTransferPlural: 'TRANSACTIES',
+  sendAsset: 'asset',
+  sendAssets: 'assets',
+  sendRecipient: 'ontvanger',
+  sendRecipients: 'ontvangers',
+  sendAssetCapital: 'Asset',
+  sendAssetsCapital: 'Assets',
+  sendRecipientCapital: 'ONTVANGER',
+  sendRecipientsCapital: 'ONTVANGERS',
+  sendCompleteNotification:
+    'Transactie in behandeling! Je saldo wordt automatisch bijgewerkt zodra de blockchain het heeft verwerkt.',
+  sendSelectAssets:
+    '{transferCount, number} van {maxNumberOfRecipients, number} Ontvangers',
+  sendTransferMessage:
+    '{transferCount, number} {transferCount, plural, one {sendTransfer} other {sendTransferPlural}} pending',
```

Comment:
"one", "other", "pending"

Reply:
One and other are operators, pending is changed.